benchmarks/setup.rb

```diff
@@ -39,7 +39,7 @@ def tags
 
 def users_with_combined_tasks
   @users_with_combined_tasks ||= rom.relation(:users).as(:user_with_combined_tasks)
-    .combine(rom.relation(:tasks).for_users)
+                                 .combine(rom.relation(:tasks).for_users)
```

Comment:
I can fix this, and update the rubocop config to stop reporting these offenses.

Want me to create a new pull-request? add a commit to this one?